services/sidecar/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM python:3.6-alpine as base
+ARG PYTHON_VERSION="3.6"
+FROM python:${PYTHON_VERSION}-slim as base
```

Comment:
Can perhaps change to normal Debian Buster if all our services use the same image? https://pythonspeed.com/articles/base-image-python-docker-images/

Reply:
so as far as I understand, using the bigger image will maybe reduce disk space in the image registry (dockerhub), maybe something also on the machines but not so sure how much. having a smaller image is for sure faster to download and to run.
I'll do some tests to check that with some numbers.